protocols/virtualfund/virtualfund-single-hop_test.go

```diff
@@ -171,24 +149,58 @@ func TestSingleHopVirtualFund(t *testing.T) {
 			}
 
 			expectedSignedState := state.NewSignedState(o.V.PreFundState())
-			aliceSig, _ := o.V.PreFundState().Sign(my.privateKey)
-			_ = expectedSignedState.AddSignature(aliceSig)
+			mySig, _ := o.V.PreFundState().Sign(my.privateKey)
+			_ = expectedSignedState.AddSignature(mySig)
 
-			forBob := protocols.Message{To: bob.address, ObjectiveId: o.Id(), SignedStates: []state.SignedState{expectedSignedState}}
-			forIrene := protocols.Message{To: p1.address, ObjectiveId: o.Id(), SignedStates: []state.SignedState{expectedSignedState}}
-			want := protocols.SideEffects{MessagesToSend: []protocols.Message{forIrene, forBob}}
+			want := protocols.SideEffects{MessagesToSend: []protocols.Message{}}
+			switch my.role {
+			case 0:
+				{
```

Comment:
I think a helper function that checks if a signed state is included in a message would DRY this out a lot.